docs/uninstall-psql.md

```diff
@@ -1,13 +1,119 @@
 # Uninstalling ProxySQL 2.x.x
 
-Uninstalling ProxySQL 2.x on Debian or Ubuntu
+You can uninstall *ProxySQL 2.x.x* using the command line.
 
-```bash
-$> sudo apt-get remove proxysql2
+## Uninstalling using APT in Debian-based Linux distributions
+
+The Advanced Package Tool (APT) handles the software installation and removal on Debian and Debian-based Linux distributions.
+
+### Listing package contents using APT
+
+To find information before uninstalling or removing the `ProxySQL2` package, you can use the following command:
```

Comment:
you can use the following command: -> use the following command

Reply:
done